### PR TITLE
Color observations black when inactive

### PR DESCRIPTION
**Pull request by oysteoh**

### CODE REVIEW

File: ertviz/controllers/multi_response_controller.py [hunk 1]
```diff
@@ -49,19 +49,26 @@ def _get_observation_plots(observation_df, x_axis):
     data = observation_df["values"]
     stds = observation_df["std"]
     x_axis = observation_df["x_axis"]
+    active_mask = observation_df["active"]
+
+    style = assets.ERTSTYLE["response-plot"]["observation"]
+    color = [style["color"] if active else "rgb(0, 0, 0)" for active in active_mask]
```
Comment (xjules) [inline ertviz/controllers/multi_response_controller.py:55]: shall we use other color (some lighter tone of red) than black?

File: tests/plots/test_controller.py [hunk 2]
```diff
@@ -26,6 +26,7 @@ def test_observation_plot_representation():
             "values": [2.85325093, 7.20311703, 21.38648991, 31.51455593, 53.56766604],
             "std": [0.1, 1.1, 4.1, 9.1, 16.1],
             "x_axis": [0, 2, 4, 6, 8],
+            "active": [True, False, True, False, False],
```
Comment (xjules) [inline tests/plots/test_controller.py:29]: Can you make it part of the `assert` test / section too, ie. testing text values as plot property?

File: ertviz/controllers/multi_response_controller.py (hunk 1 above)
Comment (oysteoh) [reply ertviz/controllers/multi_response_controller.py:56]: We can - i tried lighter red but i didn't get it to "stand out" as a deactivated one.... tried grey as well - but it is kind of difficult.... you have any suggestions?

Comment (xjules) [reply ertviz/controllers/multi_response_controller.py:56]: Did some experiments: `rgb(255,127,0)` looks pretty decent.
image

Comment (oysteoh) [reply ertviz/controllers/multi_response_controller.py:56]: agree - stand out at least! But - does it communicate `inactive` or `deactivated`?  🤔  From the look of it i cannot interpret if it is an active obs or not. With black / grey / faded / more transparent my brain immediately thinks something is turned off...

**Review by xjules — APPROVED** [review]
Bra job! :rocket:

**Pull request merged**